site/en/blog/memory-inspector-extended-cpp/index.md

```diff
@@ -5,6 +5,8 @@ authors:
   - michalpitr
 description: >
     Learn how we improved debugging C/C++ WebAssembly apps with the Memory Inspector.
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/UWLghnj8M1chllDY0Iir.jpg'
+alt: ''
```

Comment:
Looks like the images has text in it, so the alt value here should probably at least include that text.